pkg/datapath/linux/routing/routing.go

```diff
@@ -195,6 +196,44 @@ func Delete(ip net.IP, compat bool) error {
 	return nil
 }
 
+// SetupRules installs routing rules based on the passed attributes. It accounts
+// for option.Config.EgressMultiHomeIPRuleCompat while configuring the rules.
+func SetupRules(from, to *net.IPNet, mac string, ifaceNum int) error {
+	var (
+		prio    int
+		tableId int
+	)
+
+	if option.Config.EgressMultiHomeIPRuleCompat {
```

Comment:
If we converge the egress rule creation with `Configure()`, then we can leverage the fact that `Configure()` already takes in a `compat` bool which represents this option, so we can get rid of this line and replace it with:

```
if compat {
```